### ISSUE TITLE
Query strings cause serialising issues

### ISSUE THREAD

**Issue by andywooyay** [8p/GuzzleBundle#7]
If I run:

`$response = $client->get('hello');`

Everything works correctly, but if I add a query string:

~~~
$response = $client->get('hello?a=1');
// or
$response = $client->get('hello', ['query' => ['a' => '1']]);
~~~

Everything works correctly but the Symfony profiler chokes and throws:

`Exception in DataCollector.php line 35: Serialization of 'Closure' is not allowed`. 

I'm having comment out `$this->addLogs($messages);` in `\EightPoints\Bundle\GuzzleBundle\DataCollector\HttpDataCollector` to avoid this.

**Comment by florianpreusner** [comment]
Which version have you used? I'm not able to reproduce by running dev-master...

**Comment by andywooyay** [comment]
I used `composer require`, so ~2.1 with Symfony 2.6.6.

**Comment by florianpreusner** [comment]
I have created a new symfony project with guzzle bundle version 2.1.0. I'm getting different issues. Using dev-master seems to work without any problems. I will create a new version this weekend.

**Comment by ip512** [comment]
I have the same kind of issue on all forms when a file is uploaded, even if I don't use Guzzle.
I commented $this->addRequest($request, $response); in the Datacollector to avoid the issue

**Comment by florianpreusner** [comment]
Changed the behavior for collecting debug information. It would be great if you can test again using dev-master. If everything is fine I will create a new tag/version. Thanks.

**Comment by ip512** [comment]
It seems to works fine with dev-master, thank you for the fix

**Comment by florianpreusner** [comment]
Released 2.1.1.
Thanks for your contribution and testing!
